plugins/web/opentelemetry-instrumentation-user-interaction/README.md

```diff
@@ -82,6 +84,69 @@ function getData(url) {
 
 ```
 
+### Send spans for different events
+
+By default, only `click` events are automatically instrumented. To automatically instrument other events, specify the events that should be captured for telemetry. Most [browser events](https://developer.mozilla.org/en-US/docs/Web/Events) are supported.
```

Comment:
I would clarify that this applies only to events of the HTMLElement interface and its implementations.